tests/test_build_latex.py

```diff
@@ -297,3 +297,16 @@ def test_contentsname_with_language_ja(app, status, warning):
     print(status.getvalue())
     print(warning.getvalue())
     assert '\\renewcommand{\\contentsname}{Table of content}' in result
+
+
+@with_app(buildername='latex', testroot='image')
+def test_image_in_section(app, status, warning):
+    app.builder.build_all()
+    result = (app.outdir / 'Python.tex').text(encoding='utf8')
+    print(result)
+    print(status.getvalue())
+    print(warning.getvalue())
```

Comment:
"No good reason" is a bit harsh. The prints don't harm in the success case since stdout is suppressed in a normal test run, and are useful in the error case. And I think it even goes back further, I put a few of these in myself.